deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -106,32 +122,36 @@ def _get_merge_task_options(
                 str(hb_idx)
             ]
             for entry_index in range(entry_start, entry_end):
-                entry = compacted_delta_manifest.entries[entry_index]
+                entry: ManifestEntry = compacted_delta_manifest.entries[entry_index]
 
-                current_entry_size = estimate_manifest_entry_size_bytes(
+                current_entry_size: Optional[
+                    float
+                ] = estimate_manifest_entry_size_bytes(
                     entry=entry,
                     operation_type=OperationType.PYARROW_DOWNLOAD,
                     estimate_resources_params=estimate_resources_params,
                 )
-                current_entry_rows = estimate_manifest_entry_num_rows(
+                current_entry_rows: Optional[int] = estimate_manifest_entry_num_rows(
                     entry=entry,
                     operation_type=OperationType.PYARROW_DOWNLOAD,
                     estimate_resources_params=estimate_resources_params,
                 )
```

Comment:
```suggestion
                ) or 0
```

Is this better than multiple if-else below?